doc/snek.adoc

```diff
@@ -540,8 +539,8 @@ Lists and Tuples, Dictionaries are not indexed by
 numbers. Instead, Dictionaries are indexed by another Snek
 value. The only requirement is that the index value be immutable,
 so that it can never change. Lists and Dictionaries are the only mutable data
-structures in Snek, so there are lots of options for
-Dictionary indices.(((Dictionary)))
+structures in Snek: anything else can be used as a
+Dictionary index.(((Dictionary)))
```

Comment:
it's a bit more subtle than this -- you can't use a tuple containing a dict or list (the check is recursive), but I think this is probably fine